### PR TITLE
Generalize passing psd and fit cfgs to Noise operation

### PR DESCRIPTION
**Pull request by BrianJKoopman**
I tested this with this `process_pipe`:
```
process_pipe:
    - name : "psd"
      signal: "signal" # optional
      wrap: "psd" # optional
      process:
        psd_cfgs: # optional, kwargs to scipy.welch
          nperseg: 1024
        wrap_name: "psd" # optional
      calc: True
      save: True
    - name: "noise"
      calc:
        low_f: 5
        high_f: 10
      save: True
      select:
        max_noise: 2000
```

The changes below were mostly in response to each failed run of the pipeline. I think this should probably be tested with a more complicated "noise" config before merging though.

### CODE REVIEW

File: sotodlib/preprocess/processes.py [hunk 1]
```diff
@@ -330,50 +330,66 @@ class Noise(_Preprocess):
...
+                                                    **self.calc_cfgs)
         else:
             wn = tod_ops.fft_ops.calc_wn(aman, pxx=psd.Pxx,
                                          freqs=psd.freqs,
-                                         **self.calc_cfgs['noise_args'])
+                                         **self.calc_cfgs)
             noise = core.AxisManager(aman.dets)
             noise.wrap("white_noise", wn, [(0,"dets")])
-            if self.calc_cfgs['wrap_name'] is None:
+            if self.calc_cfgs.get('wrap_name') is None:
```
Comment (msilvafe) [inline sotodlib/preprocess/processes.py:373]: This if-else block is indented once to much. It should get wrapped whether or not fit = True.

File: sotodlib/preprocess/processes.py [hunk 2]
```diff
@@ -330,50 +330,66 @@ class Noise(_Preprocess):
...
+
+     - name: "noise"
+       calc:
+         low_f: 5
+         high_f: 10
+       save: True
+       select:
+         max_noise: 2000
+
     .. autofunction:: sotodlib.tod_ops.fft_ops.calc_wn
```
Comment (msilvafe) [inline sotodlib/preprocess/processes.py:344]: This function is only called if fit = False, if True, fit_noise_model is called.

**Review by msilvafe — CHANGES_REQUESTED** [review]
A couple changes then ready to merge I think.

File: sotodlib/preprocess/processes.py (hunk 1 above)
Comment (BrianJKoopman) [reply sotodlib/preprocess/processes.py]: Fixed in [703905e](https://github.com/simonsobs/sotodlib/pull/749/commits/703905e3bcb1b32f1e428ed739071455e27b3f98).

File: sotodlib/preprocess/processes.py (hunk 2 above)
Comment (BrianJKoopman) [reply sotodlib/preprocess/processes.py:344]: Want me to add the `fit_noise_model` autofunction here too then?

Comment (BrianJKoopman) [reply sotodlib/preprocess/processes.py:344]: After talking about this at lunch, I'm planning on linking to where both of these functions are documented instead. Will get to that later this afternoon.

Comment (BrianJKoopman) [reply sotodlib/preprocess/processes.py]: Alright, linked to this in [da22eb0](https://github.com/simonsobs/sotodlib/pull/749/commits/da22eb0502d83efea8e7dcbbd718e5c5b77478bf). I had to add an automodule block to the `tod_ops` docs page.

**Review by msilvafe — APPROVED** [review]
Looks good, thanks @BrianJKoopman

**Pull request merged**